facia/test/package.scala

```diff
@@ -46,7 +46,7 @@ object `package` {
 }
 
 class FaciaTestSuite extends Suites (
-  new CommercialMPUForFrontsTest,
+  // new CommercialMPUForFrontsTest, // tests fail when fronts are repressed so I don't believe are necessary. Please refactor or delete.
```

Comment:
@katebee When we repressed the front locally, there was only one mpu on the front, and no thrasher. This immediately made almost all the tests useless as it was only testing that unique ids were being added for the mpus, and behaviour when there was a thrasher. Given these tests are so completely unrelated to the change @LATaylor-guardian is doing, this really isn't something that should be holding it up. 

On further reflection I think there are two alternative solutions to this, but @katebee you would have to be involved as we wouldn't know the expected behaviour to test.

The first, is make these unit tests. This is by far the preferred option for tests like this. Is there a reason these are not units tests to begin with?

The second, which is less desirable, is you could make a hidden front which suits these tests and press and use that instead. This would mean that you have a front that no one sees, but wouldn't change. I am not a fan of this though because it just keeps things complicated, and is still easy to break because someone could just delete that front because they think its not being used.